src/messages/IPostMessageSentHandler.ts

```diff
@@ -6,14 +6,14 @@ import { IMessage } from './IMessage';
 export interface IPostMessageSentHandler {
     /**
      * First step when a handler is executed: Enables the handler to signal
-     * to the Rocketlets framework whether the it shall actually execute for the message
+     * to the Rocketlets framework whether handler shall actually execute for the message
      * about to be sent.
      *
      * @param message The message which is being sent
      * @param read An accessor to the environment
      * @return true: run the pre-logic
      */
-    isMessageApplicable(message: IMessage, read: IRead): boolean;
+    checkPostMessageSent(message: IMessage, read: IRead): boolean;
```

Comment:
@graywolf336 naming things: I thought `check + InterfaceName` is more intuitive and unique at the same time